src/pytorch_lightning/strategies/colossalai.py

```diff
@@ -264,23 +266,50 @@ def setup_precision_plugin(self) -> None:
                 )
         assert isinstance(self.model, (pl.LightningModule, _LightningPrecisionModuleWrapperBase))
         pl_module = self.model
-        process_group = ProcessGroup()
+
         if not hasattr(pl_module, "_colossalai_zero"):
-            if self.use_chunk:
-                chunk_size = self.chunk_size or ChunkManager.search_chunk_size(
-                    self.model, **self.chunk_size_search_kwargs
+            if not _COLOSSALAI_GREATER_EQUAL_0_1_10:
+                if self.use_chunk:
+                    chunk_size = self.chunk_size or ChunkManager.search_chunk_size(
+                        self.model, **self.chunk_size_search_kwargs
+                    )
+                else:
+                    chunk_size = None
+                process_group = ProcessGroup()
+                chunk_manager = ChunkManager(
+                    chunk_size,
+                    process_group,
+                    self.enable_distributed_storage,
+                    GeminiManager.get_default_device(self.placement_policy),
                 )
+                gemini_manager = GeminiManager(self.placement_policy, chunk_manager)
+                model = _LightningModuleWrapperBase(self.model)
+                self.model = ZeroDDP(model, gemini_manager, self.force_outputs_fp32)
             else:
-                chunk_size = None
-            chunk_manager = ChunkManager(
-                chunk_size,
-                process_group,
-                self.enable_distributed_storage,
-                GeminiManager.get_default_device(self.placement_policy),
-            )
-            gemini_manager = GeminiManager(self.placement_policy, chunk_manager)
-            model = _LightningModuleWrapperBase(self.model)
-            self.model = ZeroDDP(model, gemini_manager, self.force_outputs_fp32)
+                with _patch_cuda_is_available():
+                    from colossalai.nn.parallel import GeminiDDP
```

Comment:
This import failed even though GPU ci installs [`0.1.10+torch1.12cu11.3`](https://dev.azure.com/Lightning-AI/lightning/_build/results?buildId=121637&view=logs&j=3f274fac-2e11-54ca-487e-194c91f3ae9f&t=46417416-7aac-584b-0305-5d8c82c23854&l=47)

Looks like it was only added in 0.11, not 0.10 as the PR implements

Reply:
yeah. `GeminiDDP` is newly added in 0.1.12.

Reply:
@1SAA then let's update the version check to `_COLOSSALAI_GREATER_0_1_12`?